src/main/java/frc/robot/commands/ShooterModeCycleUp.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.ShooterSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
```

Comment:
Let's add a short comment here on what the ShooterModeCycleUp command is doing.

If I understood, it changes the "type" of aiming that you will be doing with the shooter...Between several different positions on the field, or an automatic aiming.

Let's make it clearer with some comments